.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug to help us improve.
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide a *minimal* script to reproduce the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error Output
+      description: Please indicate the output from the above script and what you expect the output to be.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: freud Version
+      description: What version of freud are you using?
+      placeholder: v2.10.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Python Version
+      description: What version of python are you using?
+      placeholder: v3.8.2
+  - type: dropdown
+    attributes:
+      label: System Platform
+      description: What platform are you using?
+      options:
+        - MacOS
+        - Linux
+  - type: dropdown
+    attributes:
+      label: Installation method
+      description: How did you install freud?
+      options:
+        - Download from PyPI
+        - Download from conda-forge
+        - built from source
+  - type: dropdown
+    attributes:
+      label: Developer
+      description: Who should fix this?
+      options:
+        - I can!
+        - I can, but I'll need some help and guidance.
+        - Someone else should fix this.
```

Comment:
Can we reword this to something that sounds friendlier and less demanding?

Reply:
I thought this sounded friendlier than the typical "me/someone else" options, do others think this sounds demanding?

Reply:
I think that sounds nicer. Saying that someone else *should* fix it is what sounds demanding to me.